src/stratigraphy/benchmark/ground_truth.py

```diff
@@ -14,25 +13,9 @@
 class GroundTruthForFile:
```

Comment:
I don't see much use for this class anymore, shall we remove it?

Reply:
Removed.